builtin/providers/aws/resource_aws_db_instance.go

```diff
@@ -65,6 +65,13 @@ func resourceAwsDbInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"character_set_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
Is the semantic here that this is returned by the AWS API and can't be changed (in which case it probably doesn't need `ForceNew`), or that it is an input?